### ISSUE TITLE
[BUG] Connected Glass not connecting from one side

### ISSUE THREAD

**Issue by ghost** [SuperMartijn642/ConnectedGlass#34]
**Version Info**
- Minecraft, 1.12.2
- Connected Glass, 1.1.6a

**Description of the Bug**
Connected glass not looking connect frome one side, other side looks connected, see screenshots.

**Steps to Reproduce**
Use any clear glass pane/other glass from the mod 

**Screenshots**
Side looking weird :
![wrongside](https://user-images.githubusercontent.com/30569881/212479020-743270f5-9bc4-483e-bd3b-9e725431a4bd.png)

Side looking normal:
![goodside](https://user-images.githubusercontent.com/30569881/212479037-79ebada9-939d-4788-b91c-6281e219903f.png)

--
DEBUG LOG :
[debug.log](https://github.com/SuperMartijn642/ConnectedGlass/files/10418016/debug.log)

LATEST LOG : 
[latest.log](https://github.com/SuperMartijn642/ConnectedGlass/files/10418017/latest.log)

**Comment by SyluxS7** [comment]
Seems it was a mod conflicting with this one. By deleting my instance and creating a new one with only your mod it seems to work without acting weird

**Comment by SyluxS7** [comment]
Okay so the culprit for this issue seems to be Optifine but your mod worked fine with OF last time i played :/

**Comment by trekkie22** [comment]
I have the exact same problem on 1.19.2. It only happens with Optifine installed. The glass connects the wrong way facing east and south. Tried several settings in Optifine, but was unable to find one that repairs it. I first suspected Connected Textures, but no change if CT is enabled or disabled. FastMath and FastRender also does not change anything.  Edit: Tried now every Optifine setting. No change at all.
